installer/scripts/common/setupIstio.sh

```diff
@@ -1,23 +1,39 @@
 #!/bin/bash
 source ./common/utils.sh
 
-kubectl create namespace istio-system
+kubectl get ns istio-system
+ISTIO_AVAILABLE=$?
+if [[ "$ISTIO_AVAILABLE" == 0 ]] && [[ "$ISTIO_INSTALL_OPTION" == "Reuse" ]]; then
+    # An istio-version is already installed
+    print_info "Istio installation is reused but its compatibility is not checked"
+elif [[ "$ISTIO_AVAILABLE" == 0 ]] && ([[ "$ISTIO_INSTALL_OPTION" == "StopIfInstalled" ]] || [[ "$ISTIO_INSTALL_OPTION" == "" ]]); then
```

Comment:
Please also handle 
```bash
[ "$ISTIO_INSTALL_OPTION" == "ISTIO_INSTALL_OPTION_PLACEHOLDER" ]]
```
to remain compatibility with older CLIs

(not sure if it goes into this if statement or into the else branch)

Reply:
Thanks, good point! Addressed.